src/main/java/be/appwise/core/networking/proxyman/Base64TypeAdapter.kt

```diff
@@ -0,0 +1,26 @@
+package be.appwise.core.networking.proxyman
+
+import android.util.Base64
+import com.google.gson.TypeAdapter
+import com.google.gson.stream.JsonReader
+import com.google.gson.stream.JsonWriter
+import java.io.IOException
+
+
+class Base64TypeAdapter : TypeAdapter<Data?>() {
+    @Throws(IOException::class)
```

Comment:
Is this line really needed? Removing this doesn't give me a warning that something can be throwing an Exception 🤔